webprojects/amo.md

```diff
@@ -0,0 +1,93 @@
+
+The content for the Disco Pane comes from the API. To translate it, you will need to look in [AMO (addons-server)](https://pontoon.mozilla.org/projects/amo) project.
+
+### Test the Discovery Pane
+
+The strings will be available on Dev or Staging server depending on when they were committed. See [the section below](https://github.com/mozilla-l10n/localizer-documentation/blob/2f065c6a15208cf30ec0a8d440a399a5b5424d39/webprojects/amo.md#frequencies-of-newly-extracted-strings-and-landing-of-newly-localized-strings) for details.
+
+The hosts for the Disco Pane are as follows:
+
+* Dev: https://discovery.addons-dev.allizom.org
+* Staging: https://discovery.addons.allizom.org
+* Production: https://discovery.addons.mozilla.org
+
+## Frequencies of newly extracted strings and landing of newly localized strings
+
+Newly added strings are extracted every week after tagging, which is at 9am PST on Tuesdays. The strings will be on staging and ready for localization.
+
+For localized content:
+
+* Dev server: Once translations are committed into the repositories, they are visible on the dev server in 30 minutes (or as long as it takes to push the code).
+* Staging sever: the master is tagged and pushed to staging weekly on Tuesday at around 9am PST.
+* Production: the push usually starts at 9am PST every Thursday.
+
+## Advanced Configuration for testing strings from **addons-frontend**
+
+The above instruction is sufficient for basic localization testing. If you need to test strings related to the install buttons, you will need to enabled preferences to allow add-on installation to work on the development and stage servers.
+
+To fully configure Firefox to install add-ons on either dev or stage, you will
+need to do the following:
+
+* Use Firefox Nightly or Firefox Developer Edition.
+* A new profile is recommended since changing the dev certs will mark existing add-on installs as invalid (this change is not immediate).
+* Go to `about:config` and set `extensions.webapi.testing` as a Boolean to `true`.
+* To install an add-on, you will need to be able to install a signed add-on from dev or stage. Create the `xpinstall.signatures.dev-root` pref in `about:config` as a Boolean and set it to `true`.
+* For Disco Pane: Go to `about:config` and change the host in `extensions.webservice.discoverURL` to “https://discovery.addons-dev.allizom.org/“ for Dev server or “https://discovery.addons.allizom.org/“ for Staging server.
+* If you see warnings when trying to install a theme or theme previews doesn’t work, go to `about:preferences#security` and add an exception for dev or stage. You can do so by  clicking the `Exceptions` button after selecting Privacy & Security on left panel.
```

Comment:
doesn't work -> don't work
double space before clicking

Add an exception for what, where? I would drop the first instructions with about:preferences#security, either use italic or backticks on the panel name later.

Reply:
I will learn more on this after Stuart returns on Monday.

If this is beyond what an average localizer test, we can drop it.  Also some of the links don't work. They worked and then they don't again. Not sure if this has something to do with him on vacation.